Silicon/ElkhartlakePkg/Include/FspUpd.h

```diff
@@ -1,7 +1,30 @@
 /** @file
 
 Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
-SPDX-License-Identifier: BSD-2-Clause-Patent
+

```

Comment:
I think the header needs to change to BSD-2 license.

Reply:
Done